spec/support/scim_rails_config.rb

```diff
@@ -2,7 +2,13 @@
 # is included here because it is essentially a spec helper
 
 ScimRails.configure do |config|
+  # We shouldn't have to specify company and should eventually remove this.
   config.basic_auth_model = "Company"
+
+  # Note: Setting scim variables in heroku params
+  # config.subdomain - ENV['SCIM_USERNAME']
```

Comment:
I think I would prefer to pass the ENV variables into the config but I wasn't sure how to do that and test the code utilizing these configurations. Env variables proved easier to test.

Certainly a point of improvement.